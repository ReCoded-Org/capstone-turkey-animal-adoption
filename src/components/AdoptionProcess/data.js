export let adoptionProcess = [
  {
    id: 1,
    img:
      "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
    text: "Find Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 2,
    img:
      "https://i.pinimg.com/736x/5f/80/85/5f80854fd1475958717a19e345695942.jpg",
    text: "Know Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 3,
    img:
      "https://i.pinimg.com/736x/2e/23/d3/2e23d3827c8293c27cea544a8acbcd59.jpg",
    text: "Take Your Pet Home",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
];
