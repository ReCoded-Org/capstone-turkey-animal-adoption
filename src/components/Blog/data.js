export let blogContent = {
      firstArticle : {
        id: 1,
        img: "https://i.redd.it/l5tybjlw58w41.jpg",
        blogTitle: "Average Cat Weight – How to Tell if Your Cat is Overweight",
        author: "Bertram Jensen",
        publishedDate: "21 HOURS AGO",
        blogDescription: "Is there such a thing as average cat weight? According to Purina’s experts, an average weight or range will vary from breed to breed and even between males and females. Learn why here, along with what they suggest using instead.....",
        authorWork: "Author at Panoply Store",
        authorImg: "https://images1.penguinrandomhouse.com/author/223095"
      },
      secondArticle : {
        id: 2,
        img: "https://i.redd.it/l5tybjlw58w41.jpg",
        blogTitle: "Four new breeds have been recognized by the American Kennel Club (AKC)",  
        author: "Bertram Jensen",
        publishedDate: "21 HOURS AGO",
        blogDescription: "Is there such a thing as average dog weight? According to Purina’s experts, an average weight or range will vary from breed to breed and even between males and females. Learn why here, along with what they suggest using instead.....",
        authorWork: "Author at Panoply Store",
        authorImg: "https://images1.penguinrandomhouse.com/author/223095"
      },
      thirdArticle : {
        id: 3,
        img: "https://i.redd.it/l5tybjlw58w41.jpg",
        blogTitle: "Four new breeds have been recognized by the American Kennel Club (AKC)",
        author: "Bertram Jensen",
        publishedDate: "21 HOURS AGO",
        blogDescription: "Is there such a thing as average dog weight? According to Purina’s experts, an average weight or range will vary from breed to breed and even between males and females. Learn why here, along with what they suggest using instead.....",
        authorWork: "Author at Panoply Store",
        authorImg: "https://images1.penguinrandomhouse.com/author/223095"
      }
  }