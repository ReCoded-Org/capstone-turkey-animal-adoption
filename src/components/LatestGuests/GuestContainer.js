import React from "react";
import GuestItems from "./GuestItems";
import { Container, Row, Col, Carousel } from "react-bootstrap";
import "./LatestGuests.css";

const GuestContainer = ({ latestAddedGuests }) => {
  return (
    <Container>
      <Row className="guestsContainer py-5">
        <Col sm={12}>
          {latestAddedGuests && (
            <>
              <h1>{latestAddedGuests.latestGuestsTitle.title}</h1>
              <h6>{latestAddedGuests.latestGuestsTitle.subTitle}</h6>
              <Row className="mt-5 items">
                  {latestAddedGuests.latestGuestsItems.map((guest) => (
                    <GuestItems
                      key={guest.id}
                      img={guest.img}
                      buttonText={guest.buttonText}
                      petName={guest.petName}
                      petAge={guest.petAge}
                      petBreed={guest.petBreed}
                    />
                  ))}
              </Row>
            </>
          )}
        </Col>
      </Row>

      <Carousel style={{ display: "none" }} className="carousel">
        {latestAddedGuests && latestAddedGuests.latestGuestsItems.map(guest => (
          <Carousel.Item>
            <img
              className="d-block w-100"
              src={guest.img}
              alt="slideImage"
            />
          </Carousel.Item>
        ))}
      </Carousel>
    </Container>
  );
};

export default GuestContainer;