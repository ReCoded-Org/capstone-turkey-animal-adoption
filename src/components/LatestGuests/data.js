export let latestAddedGuests = {
  latestGuestsTitle: {
    title: "Latests Guests",
    subTitle: "Meet Our Latest Friends üêæ",
  },
  latestGuestsItems: [
    {
      id: 4,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Cookie",
      petAge: "2 years",
      petBreed: "Poodle",
    },
    {
      id: 5,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Alex",
      petAge: "5 years",
      petBreed: "Terrier",
    },
    {
      id: 6,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Sushi",
      petAge: "1 years",
      petBreed: "Maltese-Terrier",
    },
    {
      id: 7,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Bambi",
      petAge: "3 months",
      petBreed: "Golden Retriever",
    },
    {
      id: 8,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Noodle",
      petAge: "2 years",
      petBreed: "Pomeranian",
    },
    {
      id: 9,
      img:
        "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
      buttonText: "View",
      petName: "Princess",
      petAge: "1 years",
      petBreed: "Chihuahua",
    },
  ],
};
