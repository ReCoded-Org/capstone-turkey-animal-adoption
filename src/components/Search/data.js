export let searchGuests = {
    searchGuestsTitle: {
      title: "Adopt Your New Pet",
    },
    guestsResults: [
      {
        id: 1,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 5,
        breed: "Poodle",
        gender: "Female",
        name: "Sushi",
      },
      {
        id: 2,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 1,
        breed: "Terrier",
        gender: "Male",
        name: "Sushi",
      },
      {
        id: 3,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 2,
        breed: "Poodle",
        gender: "Female",
        name: "Sushi",
      },
      {
        id: 4,
        img:
          "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
        location: "Istanbul",
        age: 5,
        breed: "Poodle",
        gender: "Male",
        name: "Sushi",
      },
    ],
    locationCities : [
      "Istanbul",
      "Ankara",
      "Izmir"
    ],
    searchBreed : [
      "Poodle",
      "Terrier",
      "Rottweiler",
      "Golden Retriever",
      "Cat",
      "Tobby cat",
      "Scottish"
    ],
    searchAge : [
      "<3",
      ">3",
      "All"
    ],
    searchGender : [
      "Female",
      "Male",
    ],
  };