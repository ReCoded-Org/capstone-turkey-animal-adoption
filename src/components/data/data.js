export let adoptionProcess = [
  {
    id: 1,
    img:
      "https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg",
    text: "Find Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 2,
    img:
      "https://i.pinimg.com/736x/5f/80/85/5f80854fd1475958717a19e345695942.jpg",
    text: "Know Your Pet",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
  {
    id: 3,
    img:
      "https://i.pinimg.com/736x/2e/23/d3/2e23d3827c8293c27cea544a8acbcd59.jpg",
    text: "Take Your Pet Home",
    firstDescription: "Lorem Ipsum Lorem Ipsum Lorem",
    secondDescription: "Lorem Ipsum Lorem Ipsum Lorem",
  },
];

export let sectionTitle = {
  title: "Pet Adoption Process",
};

export let guestsTitle = {
  title: "Latests Guests",
  subTitle:
    "Home is behind, the world ahead and there are many paths to tread through shadows to the edge.",
};

export let guests = [
  {
    id: 4,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
  {
    id: 5,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
  {
    id: 6,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
  {
    id: 7,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
  {
    id: 8,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
  {
    id: 9,
    img:
      "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
    buttonText: "View",
  },
];