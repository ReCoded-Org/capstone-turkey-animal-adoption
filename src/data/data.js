export let guestsTitle = {
    title : "Latests Guests",
    subTitle: "Home is behind, the world ahead and there are many paths to tread through shadows to the edge."
  }
  
export let guests = [
    {
      id: 4,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
    {
      id: 5,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
    {
      id: 6,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
    {
      id: 7,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
    {
      id: 8,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
    {
      id: 9,
      img: "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F37%2F2020%2F09%2F22%2F50-cute-dog-names.jpg",
      buttonText : "View"
    },
  ];