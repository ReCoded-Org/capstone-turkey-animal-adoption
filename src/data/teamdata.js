import React from "react";
import ayse from "../images/ayse.jfif";
import rahaf from "../images/rahaf.jpg";
import halit from "../images/halit.jfif";
import mustafa from "../images/mustafa.png";
import { FaGithub } from "react-icons/fa";

export let adoptionProcess = {
  adoptionProcessTitle: "Meet Our Great Team",
  adoptionProcessItems: [
    {
      id: 1,
      img: `${ayse}`,
      text: "Ayşe Çimen Başar",
      firstDescription: "FrontEnd Engineer",
      secondDescription: (
        <a href="https://github.com/ayse8888" target="_blank">
          <FaGithub size={30} style={{ color: "#777777" }} />
        </a>
      ),
    },
    {
      id: 2,
      img: `${rahaf}`,
      text: "Rahaf Shora",
      firstDescription: "FrontEnd Engineer",
      secondDescription: (
        <a href="https://github.com/Rofy87" target="_blank">
          <FaGithub size={30} style={{ color: "#777777" }} />
        </a>
      ),
    },
    {
      id: 3,
      img: `${mustafa}`,
      text: "Mustafa Şükrü Güldağ",
      firstDescription: "FrontEnd Engineer",
      secondDescription: (
        <a href="https://github.com/mustafaguldag" target="_blank">
          <FaGithub size={30} style={{ color: "#777777" }} />
        </a>
      ),
    },
    {
      id: 4,
      img: `${halit}`,
      text: "Halit Batur",
      firstDescription: "Lead Engineer",
      secondDescription: (
        <a href="https://github.com/halitbatur" target="_blank">
          <FaGithub size={30} style={{ color: "#777777" }} />
        </a>
      ),
    },
  ],
};
