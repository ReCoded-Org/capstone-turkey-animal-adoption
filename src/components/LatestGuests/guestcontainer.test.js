import React from "react";
import renderer from "react-test-renderer";
import GuestContainer from "./GuestContainer";
import { guests, guestsTitle } from "../data/data";

it("renders correctly", () => {
  const tree = renderer
    .create(<GuestContainer guests={guests} guestsTitle={guestsTitle} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
