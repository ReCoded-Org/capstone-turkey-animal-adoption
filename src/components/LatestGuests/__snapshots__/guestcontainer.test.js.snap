// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Guest Container correctly 1`] = `
<div
  className="container"
>
  <div
    className="guestsContainer py-5 row"
  >
    <div
      className="col-sm-12"
    >
      <h1 />
      <h6 />
      <div
        className="mt-5 carouselItems row"
      >
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
        <div
          className="mb-5 col-lg-4"
        >
          <div
            className="guestItems"
            style={
              Object {
                "backgroundImage": "url(https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg)",
              }
            }
          />
          <button
            className="shadow-none btn btn-primary"
            disabled={false}
            id="viewButton"
            onClick={[Function]}
            type="button"
          >
            View
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="carousel carousel slide"
    onKeyDown={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "display": "none",
      }
    }
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="active carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="slideImage"
          className="d-block w-100 sliderImg"
          src="https://pyxis.nymag.com/v1/imgs/c9c/2c2/be941ee1a78099dc7f3efea9dab847e011-25-rescue-dog-lede.rsquare.w1200.jpg"
        />
      </div>
    </div>
    <a
      className="carousel-control-prev"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
